### PR TITLE
PSP-9710, PSP-9313: UI/UX Cleanup - Make icons consistent across PIMS

### PR DESCRIPTION
**Pull request by stairaku**
image
image
image
image

### CODE REVIEW

File: source/frontend/src/components/common/buttons/RemoveButton.tsx [hunk 1]
```diff
@@ -26,6 +31,22 @@ export const RemoveButton: React.FunctionComponent<React.PropsWithChildren<IRemo
   );
 };
 
+export const RemoveIconButton: React.FunctionComponent<
+  React.PropsWithChildren<IRemoveButtonProps>
+> = ({ onRemove, title, icon, dataTestId, style }) => {
+  return (
+    <StyledRemoveIconButton
+      variant="primary"
+      title={title ?? 'edit'}
```
Comment (asanchezr) [inline source/frontend/src/components/common/buttons/RemoveButton.tsx:40]: default title for remove buttons should be "remove" - not "edit"

File: source/frontend/src/components/common/buttons/ViewButton.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+export const ViewButton: React.FunctionComponent<React.PropsWithChildren<IViewButtonProps>> = ({
+  onClick,
+  title,
+  icon,
+  dataTestId,
+  style,
+}) => (
+  <StyledIconButton
+    variant="primary"
+    title={title ?? 'edit'}
```
Comment (asanchezr) [inline source/frontend/src/components/common/buttons/ViewButton.tsx:24]: View button default title should not be "edit" - maybe "view" would be more suitable?

File: source/frontend/src/components/common/form/ToggleSaveInput/ToggleSaveInputView.tsx [hunk 3]
```diff
@@ -65,9 +66,20 @@ export const ToggleSaveInputView: React.FC<IToggleSaveInputViewProps> = ({
   } else {
     return (
       <>
-        {asCurrency ? formatMoney(Number(value)) : value}
-        <EditButton onClick={() => setIsEditing(true)} />
+        <StyledCompensationContainer>
+          {asCurrency ? formatMoney(Number(value)) : value}
+          <EditButton onClick={() => setIsEditing(true)} />
+        </StyledCompensationContainer>
```
Comment (asanchezr) [inline source/frontend/src/components/common/form/ToggleSaveInput/ToggleSaveInputView.tsx:72]: **nit:** This component is generic so the styled container shouldn't be named "StyledCompensationContainer" - maybe just "StyledContainer". This is very minor of course

File: source/frontend/src/components/contact/ContactManagerView/ContactResultComponent/columns.tsx [hunk 4]
```diff
@@ -131,23 +130,17 @@ const columns: ColumnWithProps<IContactSearchResult>[] = [
...
               onClick={() => history.push(`/contact/${props.row.original.id}/edit`)}
-            >
-              <FaEdit size={22} />
-            </StyledIconButton>
+            />
           )}
 
           {hasClaim(Claims.CONTACT_VIEW) && (
-            <StyledIconButton
+            <ViewButton
```
Comment (asanchezr) [inline source/frontend/src/components/contact/ContactManagerView/ContactResultComponent/columns.tsx:140]: thanks for making these!

File: source/frontend/src/features/notes/list/NoteResults/columns.tsx [hunk 5]
```diff
@@ -52,26 +51,23 @@ export function createTableColumns(
...
-              <StyledRemoveLinkButton
-                title="Delete Note"
-                variant="light"
-                onClick={() => onDelete(cellProps.row.original)}
-              >
-                <FaTrash size="2rem" />
-              </StyledRemoveLinkButton>
-            )}
+            <Col xs={1} className="p-0">
+              {' '}
```
Comment (asanchezr) [inline source/frontend/src/features/notes/list/NoteResults/columns.tsx:55]: no sure whether we need this here?

**Review by asanchezr — APPROVED** [review]
Approved with comments

File: source/frontend/src/components/common/buttons/ViewButton.tsx [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+import { FaEye } from 'react-icons/fa';
+import { CSSProperties } from 'styled-components';
+
+import { StyledIconButton } from './IconButton';
+
+interface IViewButtonProps extends ButtonProps {
+  onClick: () => void;
+  title?: string;
+  icon?: React.ReactNode;
+  dataTestId?: string | null;
```
[the remark this answers is not in the document]
Comment (stairaku) [reply source/frontend/src/components/common/buttons/ViewButton.tsx:11]: Thank you for the observation. Changing the data-testId pattern on both RemoveButton and ViewButton files.

File: source/frontend/src/features/notes/list/NoteResults/columns.tsx (hunk 5 above)
Comment (stairaku) [reply source/frontend/src/features/notes/list/NoteResults/columns.tsx:55]: good catch! I forgot to clean that up!

**Pull request merged**